bellows/ezsp.py

```diff
@@ -31,17 +32,30 @@ def connect(self, device):
     def reset(self):
         return self._gw.reset()
 
+    @asyncio.coroutine
+    def version(self, version):
+        result = yield from self._command('version',version)
```

Comment:
Space after comma between arguments, please. In general, please run pep8 or flake8 against your code. You can ignore line length limits, as long as things are sensible.